gix-commitgraph/tests/fixtures/generation_number_overflow.sh

```diff
@@ -13,9 +13,25 @@ function tick () {
   export GIT_COMMITTER_DATE GIT_AUTHOR_DATE
 }
 
-tick
-function commit() {
-  local message=${1:?first argument is the commit message}
+function force_tag() {
```

Comment:
Maybe this should be named something else. Calling it `force_tag` makes it sound like it does something similar to `git tag -f`, when actually:

- `force_tag` tries (though not very hard) to avoid resetting an existing tag.
- `git tag -f` will still refuse to create a tag in the scenarios where this is intended to be called (or I would've just used `git tag -f`).